services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -484,11 +501,24 @@ async def _execute_query():
                 if exists is None:
                     ins = file_meta_data.insert().values(**vars(fmd))
                     await conn.execute(ins)
+                return fmd.file_size, fmd.last_modified
 
-        await _execute_query()
+        file_size, last_modified = await _execute_query()
 
         bucket_name = self.simcore_bucket_name
         object_name = file_uuid
+
+        # a parallel task is tarted which will update the metadata of the updated file
+        # once the update has finished.
+        asyncio.ensure_future(
```

Comment:
please use
fire_and_forget_task
from https://github.com/ITISFoundation/osparc-simcore/blob/681fc9c41e7ef8d8616050349ac5d03669d507a5/packages/service-library/src/servicelib/utils.py#L49
in case of exception it will log it. if not this goes in the void.

Reply:
I really like this one